backend/models/listing.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const ListingSchema = new mongoose.Schema({
+    listerID: {type: mongoose.Schema.Types.ObjectId, required: true},
+    name: {type: String, required: true, minlength: 5, maxlength: 30},
+    description: {type: String},
+    rental_price: {type: Number, required: true},
+    date: {type: Date},
+    pet_friendly: {type: Boolean},
+    active: {type: Boolean, default: true},
+    location: {type: { type: String }, coordinates: [Number]},
+});
+
+const Listing = mongoose.model("Listing", ListingSchema);
+module.exports = Listing;
```

Comment:
Side note, if yall are using VSCode, turn on the "Insert Final Newline" setting if you haven't already. Then this error won't show (I forgot to turn it on myself!) @dlalaj @MphiGithub @FarihaIS :D
![image](https://github.com/FarihaIS/VanRoomies/assets/69827283/c8be9d73-1179-480f-a8ab-2fb11ef1da15)

Reply:
Cool cool will do!